website/src/content/docs/transforms/cache.mdx.bak

```diff
@@ -1,157 +0,0 @@
-import { Callout } from '@theguild/components'
```

Comment:
This isn't rendered anywhere, I'm not sure if it's needed, and it was crashing my build.

We can always get it back from git history.